generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/grammatical/AnyOf.feature

```diff
@@ -122,73 +122,69 @@ Scenario: User requires to create a field with numbers that conform to multiple
+      | 2018-10-06T00:00:00.000 |
+      | 2018-10-07T00:00:00.000 |
+      | 2018-10-08T00:00:00.000 |
+      | 2018-10-09T00:00:00.000 |
 
-    @ignore
+  @ignore
     # failing - data is duplicated linked to issue 91
-Scenario: User requires to create a field with dates that conform to multiple sets of constraints
-       Given there is a constraint:
-       """
-       { "anyOf": [
-         { "field": "foo", "is": "after", "value": { "date": "2018-10-01T00:00:00.000" } },
-         { "field": "foo", "is": "afterOrAt", "value": { "date": "2018-10-02T00:00:00.000" } }
-       ]}
-       """
-       And there is a constraint:
-       """
-       { "anyOf": [
-         { "field": "foo", "is": "after", "value": { "date": "2018-10-03T00:00:00.000" } },
-         { "field": "foo", "is": "afterOrAt", "value": { "date": "2018-10-04T00:00:00.000" } }
-       ]}
-       """
-       And foo is of type "datetime"
-       And foo is before 2018-10-09T00:00:00.000
-     Then the following data should be generated:
-       | foo                     |
-       | null                    |
-       | 2018-10-03T00:00:00.001 |
-       | 2018-10-04T00:00:00.000 |
-       | 2018-10-05T00:00:00.000 |
-       | 2018-10-06T00:00:00.000 |
-       | 2018-10-07T00:00:00.000 |
-       | 2018-10-08T00:00:00.000 |
+  Scenario: User requires to create a field with dates that conform to multiple sets of constraints
+    Given there is a constraint:
+      """
+      { "anyOf": [
+        { "field": "foo", "is": "after", "value": { "date": "2018-10-01T00:00:00.000" } },
+        { "field": "foo", "is": "afterOrAt", "value": { "date": "2018-10-02T00:00:00.000" } }
+      ]}
+      """
+    And there is a constraint:
+      """
+      { "anyOf": [
+        { "field": "foo", "is": "after", "value": { "date": "2018-10-03T00:00:00.000" } },
+        { "field": "foo", "is": "afterOrAt", "value": { "date": "2018-10-04T00:00:00.000" } }
+      ]}
+      """
+    And foo is of type "datetime"
+    And foo is before 2018-10-09T00:00:00.000
+    Then the following data should be generated:
+      | foo                     |
+      | null                    |
+      | 2018-10-03T00:00:00.001 |
+      | 2018-10-04T00:00:00.000 |
+      | 2018-10-05T00:00:00.000 |
+      | 2018-10-06T00:00:00.000 |
+      | 2018-10-07T00:00:00.000 |
+      | 2018-10-08T00:00:00.000 |
 
-Scenario: Running an 'anyOf' request that contains a valid nested anyOf request should be successful
-  Given there is a constraint:
-    """
+  Scenario: Running an 'anyOf' request that contains a valid nested anyOf request should be successful
+    Given there is a constraint:
+      """
       { "anyOf": [
-        {
-          "field": "foo", "is": "ofLength", "value": 1
-        },
-        {
-          "anyOf":  [
-            { "field": "foo", "is": "ofLength", "value": 3 },
-            { "field": "foo", "is": "ofLength", "value": 5 }
-          ]
-        }
+        { "field": "foo", "is": "ofLength", "value": 1 },
```

Comment:
Were the changes like this and all other non-whitespace changes auto formatted for your by IntelliJ or did you do them by hand.

If the latter, maybe it should be called out in the documentation to clarify this as being the preferred style. i.e. fully-expanded for conditional constraints, 'fully-collapsed'(?) for simple/atomic constraints

Reply:
By hand. I don't know if it needs to be a strict convention, we can hopefully trust our authors to apply sensible discretion.